gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```

Comment:
Is there a reason to update the gradle version? IMO, this should be handled in a separate PR if needed

Reply:
It is possible to find a workaround, but it would be better to update it. There is a compatibility issue between ```gradle``` and ```java``` versions, without resolving it I am not able even a compile a project. I don't use the latest ```JDK```, just one which is comfortable to work with on the rest of my projects. I believe it is a time to migrate to more recent ```gradle``` version. 

I will submit a new PR with change in gradle version, but change in this PR code will be left as is.

Reply:
Please, here is a separate PR with this change: https://github.com/web3j/web3j/pull/1861